### PR TITLE
Opaque Streams

### PR DESCRIPTION
**Pull request by Maithem**
## Overview
Opaque Streams Feature

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public AddressSpaceView(@Nonnull final CorfuRuntime runtime) {
             cacheBuilder.weigher((k, v) -> (int) (CACHE_KEY_SIZE + MetricsUtils.sizeOf.deepSizeOf(v)));
         }
 
-        if (cacheDisabled) {
+        if (true) {
             cacheBuilder.maximumSize(0); // Do not allocate memory when cache is disabled.
```
Comment (xiaoqin2012) [inline runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java]: The code will never go to the else statements?

File: runtime/src/main/java/org/corfudb/protocols/logprotocol/OpaqueMultiSMREntries.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.UUID;
+
+import static org.corfudb.util.serializer.CorfuSerializer.corfuPayloadMagic;
+
+/**
+ *
+ *
+ * Created by Maithem on 2/3/20.
+ */
+public class OpaqueMultiSMREntries {
```
[the remark this answers is not in the document]
Comment (Maithem) [reply runtime/src/main/java/org/corfudb/protocols/logprotocol/OpaqueMultiSMREntries.java]: Cleaning up now.

File: runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java (hunk 1 above)
Comment (Maithem) [reply runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java]: Was testing something. Removed.

**Comment by Maithem** [comment]
obsolete.

**Pull request closed without merging**